packages/forms/src/components/Combobox/utils.ts

```diff
@@ -1,6 +1,5 @@
 import { ValidationRule } from "react-hook-form";
 import orderBy from "lodash/orderBy";
-import isArray from "lodash/isArray";
 import isNumber from "lodash/isNumber";
 import isObject from "lodash/isObject";
 import isString from "lodash/isString";
```

Comment:
I know it was not part of the scope but, we could probably get rid of these as well :thinking: 

```tsx
// isNumber
typeof x === "number"

// isObject
x instanceof Object

// isString
typeof x === "string"
```